lib/nanoc/base/repos/store.rb

```diff
@@ -34,6 +34,12 @@ def initialize(filename, version)
       @version  = version
     end
 
+    # Logic for building tmp path from active environment and store name
+    # @api private
+    def self.tmp_path_for(env, store)
```

Comment:
When calling this function, the argument names aren’t obvious. Using named arguments will make this nicer.